bundles/org.openhab.core.compat1x.test/META-INF/MANIFEST.MF

```diff
@@ -1,18 +1,19 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: openHAB 1.x Compatibility Layer Tests
-Bundle-SymbolicName: org.openhab.core.compat1x.test
-Bundle-Version: 2.2.0.qualifier
-Bundle-Vendor: openHAB
-Fragment-Host: org.openhab.core.compat1x
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Import-Package: groovy.lang,
- org.codehaus.groovy.reflection,
- org.codehaus.groovy.runtime,
- org.codehaus.groovy.runtime.callsite,
- org.codehaus.groovy.runtime.typehandling,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.core.events,
- org.eclipse.smarthome.test,
- org.hamcrest;core=split,
- org.junit
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2

```

Comment:
Please note that the order does not match the order in the archetype, should that one also be fixed otherwise we can keep formatting / sorting them after every new PR.

Reply:
Hm, the tool should change the archetype, too. Has this been missed?

Reply:
It did, I checked the master at first, did not realize your smarthome PR was not yet merged.